pkg/controller/account/account_controller.go

```diff
@@ -1256,6 +1256,21 @@ func (r *ReconcileAccount) handleCreateAdminAccessRole(
 			return nil, nil, err
 		}
 
+		_, err = createManagedOpenShiftSupportRole(
+			reqLogger,
+			awsSetupClient,
+			ccsClient,
+			adminAccessArn,
+			currentAccInstanceID,
+			tags,
+			SREAccessARN,
+		)
+
+		if err != nil {
+			reqLogger.Error(err, "createManagedOpenShiftSupportRole err", roleID)
```

Comment:
can you add something more descriptive here?

Reply:
Updated